test/utils.js

```diff
@@ -30,9 +30,10 @@ export const runQuery = (query: string, rootValue: ?any = { accessToken: null, u
  *
  * @see runQuery
  */
-export const runAuthenticatedQuery = (query: string, userID: string = "user-42") => {
-  return runQuery(query, {
-    accessToken: "secret",
-    userID,
-  })
+export const runAuthenticatedQuery = (query: string, rootValue: ?any = {}) => {
+  const authenticatedRootValue = rootValue
+  authenticatedRootValue.accessToken = "secret"
+  authenticatedRootValue.userID = "user-42"
+
+  return runQuery(query, authenticatedRootValue)
```

Comment:
A more idiomatic JS way would be to merge objects:

```js
return runQuery(query, Object.assign({ accessToken: "secret", userID: "user-42" }, rootValue))
```

This merges the 2nd object into the 1st one, a nice side-effect is that _if_ the user of the function did want to pass in custom `accessToken` or `userID` values they can just include them in the `rootValue`.